config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -93,6 +93,20 @@ message ContextSpecificConfig {
 
   // timestamp (in milliseconds since epoch) at which this config was last updated
   int64 update_timestamp = 4;
+}
+
+message InternalContextSpecificConfig {
```

Comment:
This doesn't need to be a proto, and it doesn't belong to the API. Since it's internal, we can just build a POJO containing a ContextSpecificConfig and an optional prev config.

Reply:
Done